### PR TITLE
feat: open manage app rpc

### PR DESCRIPTION
**Pull request by zachferland**

### CODE REVIEW

**Comment by zachferland** [comment]
@PaulLeCam just pushed up partial implementation i had, but waited to align with the work you mentioned you had started as well

**Comment by PaulLeCam** [comment]
Thanks! I'm wondering if we need the high-level RPC client/handler if all we need is to send a callback from the management app?
I think on the management app side, we need to use the `AuthProviderClient` so we need to instantiate it with 3ID Connect's window object. Should we get it from `window.parent` or listen to an even on the transport to know what target to use?

**Comment by zachferland** [comment]
@PaulLeCam this wires it up now, works when running `npm run start:demo`

Note, that this assumes management already build in public folder right now, and it doesnt pass ceramic endpoint (ie local vs clay) to management app build. 

Also think there may be a regression bug with linking, going to look at that tmrrw.

File: connect/src/iframeDisplay.ts [hunk 1]
```diff
@@ -81,3 +83,68 @@ export const createIframe = (iframeUrl: string): HTMLIFrameElement => {
...
+
+      await new Promise((resolve) => {
+        app.onload = () => {
+          resolve()
+        }
+      })
+      // @ts-ignore
+      const observer = createMessageObservable(window)
+
+      await new Promise(res => {
```
Comment (PaulLeCam) [inline connect/src/iframeDisplay.ts:123]: Not an issue but Rx provides APIs to do this, for example: https://github.com/ceramicnetwork/js-transports/blob/main/packages/rpc-transport/src/index.ts#L49

Comment (zachferland) [reply connect/src/iframeDisplay.ts]: ah yes, nice, changed that

**Comment by zachferland** [comment]
@PaulLeCam going to merge so that all wiring is in, is there any short term workaround for management build issue referencing 3id-connect? can it not reference a 3id-connect release instead? (did try this, had some issues) 

could switch to mono repo now if you think that is fairly easy, only concern there is making sure we dont release 3id-connect to latest (3box still there)

**Comment by PaulLeCam** [comment]
Ha I just commented on https://github.com/ceramicstudio/3id-connect/pull/44 as I was trying to fix the CI issue.
I think monorepo makes sense, just not using Lerna for publishing (doing it manually instead) to avoid possible issues, what do you think?
IMO the main thing is extracting the AuthProviderRelay (at least the client) and Manage3IDs modules so they can be used by the management app without having a dependency on the 3ID Connect module.

**Comment by zachferland** [comment]
ha thanks, will follow up there

**Pull request merged**